essentials-framework/Essentials DM/Essentials_DM/Chassis/DmChassisController.cs

```diff
@@ -1094,6 +1094,133 @@ public void ExecuteSwitch(object inputSelector, object outputSelector, eRoutingS
         }
         #endregion
 
+        #region IRoutingNumeric Members
+
+        public void ExecuteNumericSwitch(ushort inputSelector, ushort outputSelector, eRoutingSignalType sigType)

```

Comment:
Same as DmBladeChassis. Make this method call the existing ExecuteSwitch instead of duplicating it